### DISCUSSION TITLE
Is it still possible to see what codespell finds?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2809] [General]
This project uses codespell to check files that are changed and created. The results are shown by a badge on in the readme page (where I guess it to be seen quite often) and in the actions. The report that can be seen includes: 

> Codespell
> Process completed with exit code 65.

But I don't appear to be able to see what errors it found, or at least which files made it report an issue.

Can this still be seen at some place?

**Comment by isaak654** [comment]
I'm not sure how to fix the exit code, however it disappears once it no longer detects any typo after its execution.

> But I don't appear to be able to see what errors it found

https://github.com/sandboxie-plus/Sandboxie/actions/runs/4653350654/jobs/8234147951#step:3:61

![action_log](https://user-images.githubusercontent.com/12372772/230891976-dc38e455-63c4-4c9a-9e17-649441745896.png)

**Reply by bastik-1001** [reply]
All I seem when I follow the link is un-collapsed "Check spelling errors:" with the entry "Error: "

![Spell check · sandboxie-plus_Sandboxie@6a2eb42](https://user-images.githubusercontent.com/1449429/230891835-80a46fcb-d507-4023-a01f-3c02cec16a7b.png)

**Reply by isaak654** [reply]
There is the chance you could only do this on your fork or it's a browser issue.

**Reply by bastik-1001** [reply]
One might be the reason. Thank you.
